example/docker-compose.extra.yml

```diff
@@ -0,0 +1,102 @@
+# Wikibase for testing
+version: '3'
+
+services:
+  elasticsearch:
+    image: "${ELASTICSEARCH_IMAGE_NAME}"
+    restart: unless-stopped
+    networks:
+      default:
+        aliases:
+         - elasticsearch.svc
+    environment:
+      discovery.type: single-node
+      ES_JAVA_OPTS: "-Xms512m -Xmx512m"
+  wdqs-frontend:
+    image: "${QUERYSERVICE_UI_IMAGE_NAME}"
+    restart: unless-stopped
+    ports:
+     - "${WDQS_FRONTEND_PORT}:80"
+    depends_on:
+      - wdqs-proxy
+    networks:
+      default:
+        aliases:
+         - wdqs-frontend.svc
+    environment:
+      - WIKIBASE_HOST=${WIKIBASE_HOST}
```

Comment:
Does this also need the port? (Same for `wdqs-updater` below).

Reply:
I think this is fine as it's going through docker internally where WDQS_HOST is the proxy
```
    location /proxy/wikibase {
        rewrite /proxy/wikibase/(.*) /${DOLLAR}1 break;
        proxy_pass http://$WIKIBASE_HOST:80;
    }

    location /proxy/wdqs {
        rewrite /proxy/wdqs/(.*) /${DOLLAR}1 break;
        proxy_pass http://$WDQS_HOST:80;
    }
```

Reply:
same thing with wdqs-updater

Reply:
wdqs-updater "should" be fine (tm)

However looking at it I see some room for improvement at  `runUpdate.sh`

```
/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
/wait-for-it.sh "$WDQS_HOST:$WDQS_PORT" -t 300 -- \
./runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "$WDQS_ENTITY_NAMESPACES"

```

the WIKIBASE_HOST variable is re-used on multiple places making it a bit quirky in different situations.

--conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" 

Which I suppose is used for outputting ` <http://wikibase.svc/entity/Q1>`

and at the same time for -wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST

Reply:
In the example we want the updater to use the internal wikibase.svc for updating but should output `<http://localhost/entity/Q1>`

Reply:
Ok i tried updating doesn't seem to to be all that useful since it seems we would also have set WIKIBASE_HOST to the same value as WIKIBASE_CONCEPT_URI. 

If we don't do that we get the following scenario

1. concept uri set to somethingcool.svc
2. wdqs-updater goes to http://wikibase.svc to update which makes wikibase respond with <http://wikibase.svc> in EntityData. ie making this variable pointless as we would also need to switch the WIKIBASE_HOST to be the same as the concept uri.

Reply:
example -> https://github.com/wmde/wikibase-release-pipeline/pull/136

Reply:
I think I wasn't understanding your concerns. The thing is that you can set the hostname of the webservice to be "coolplace.io" anyway and docker should still route it the right place. I'm not sure if that makes sense?